### PR TITLE
Decouple form element styles

### PR DESCRIPTION
**Pull request by cherewaty**
Should help to alleviate the specificity problems we've been experiencing with the `form.css` styles.

### CODE REVIEW

File: lib/sharedStyles/form.css [hunk 1]
```diff
@@ -99,33 +94,6 @@
   }
 }
 
-/**
- * Specific <input> style
- */
-input.input,
-.input input {
```
Comment (cherewaty) [inline lib/sharedStyles/form.css]: I'm really confused as to what this selector was doing.

Comment (wwilsman) [reply lib/sharedStyles/form.css]: It was initially just `input.input`, I'm assuming for forced specificity reasons. I added the `.input input` when I moved the `.input` class to be on the wrapper.

**Comment by cherewaty** [comment]
I haven't tested with the `stripes-core` login form yet. I'm going to take another pass at this before merging - likely with some class name changes too (`input` -> `formControl`?)

**Comment by rasmuswoelk** [comment]
@cherewaty It does not fix the login screen but I just submitted a PR that will.

And I agree; we should change the class to `formControl` since the class is not only being applied to `<input>`'s.

**Comment by cherewaty** [comment]
Added commit that renames `input` to `formControl`.

**Pull request merged**